### PR TITLE
Improvements to asset list view

### PR DESCRIPTION
**Pull request by dmpl1**
### Requirements
- Improvements to Assets Page

### Description of the Change
- Environment Assets Page is updated with new UI. 
- Field mapping constant file is added. 
- Order by Pipe is added.

### CODE REVIEW

**Comment by dmpl1** [comment]
> Request body with `asset_type: undefined` resulting in empty asset responses

File: core/jazz_ui/src/app/core/constants/field-name-mapping.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+// Note: LHS should be in lowercase. 
+
+export const MAPPING = {
+    'apigateway': 'API Gateway',
```
Comment (raghits) [inline core/jazz_ui/src/app/core/constants/field-name-mapping.ts]: Update to complete list.

Comment (devsatishm) [reply core/jazz_ui/src/app/core/constants/field-name-mapping.ts]: Complete list: https://github.com/tmobile/jazz/blob/sls_app/core/jazz_assets/swagger/swagger.json#L131

Comment (devsatishm) [reply core/jazz_ui/src/app/core/constants/field-name-mapping.ts]: From the list, following assets need mapping: 
cloudwatch_event => `CloudWatch Event`
apigee_proxy => `Apigee Proxy`
kinesis_stream => `Kinesis Stream`

**Pull request merged**